src/server/middleware/proxyMiddleware.ts

```diff
@@ -56,13 +56,31 @@ const staticAssetsMiddleware = createProxyMiddleware('/static', {
   target: 'http://localhost:8081'
 });
 
-const apiProxyMiddleware = createProxyMiddleware('/api', {
-  target: 'https://staging-2020.ensembl.org',
+// const apiProxyMiddleware = createProxyMiddleware('/api', {
+//   target: 'https://staging-2020.ensembl.org',
+//   changeOrigin: true,
+//   secure: false
+// });
```

Comment:
Can this be removed?

Reply:
Ouch, forgot the dev code. I've reverted the changes to the `proxyMiddleware` file.